straxen/plugins/veto_events.py

```diff
@@ -31,9 +20,34 @@ class nVETOEvents(strax.OverlapWindowPlugin):
     data_kind = 'events_nv'
     compressor = 'zstd'
 
-    __version__ = '0.0.2'
+    __version__ = '0.0.3'
     events_seen = 0
 
+    event_left_extension_nv = straxen.URLConfig(
+        default=0,
+        track=True,
+        type=int,
+        help='Extends event window this many [ns] to the left.'
+    )
+    event_resolving_time_nv = straxen.URLConfig(
+        default=200,
+        track=True,
+        type=int,
+        help='Resolving time for window coincidence [ns].'
+    )
+    event_min_hits_nv = straxen.URLConfig(
+        default=3,
+        track=True,
+        type=int,
+        help='Minimum number of fully confined hitlets to define an event.'
+    )
+    channel_map = straxen.URLConfig(
+        default=3,
```

Comment:
Is this a typo? should be a immutabledict right?

Reply:
Ah yes good catch. Never just copy paste things. Thanks. Processing worked and setting and lineage used the correct channel map. Will leave the default empty.